src/components/member-profile/contribution/index.js

```diff
@@ -135,6 +152,13 @@ const ContributionCard = ({
           )}
         </div>
         <div className={classNames.rightSection}>
+          {isNoteworthy !== undefined ? (
```

Comment:
You can use `&&` instead of having if-else if you have only condition in which you want to conditionally render components or elements

Reply:
I want to do this because I want to render that component when noteworthy is not undefined which will be the case if the contribution is  a PR and I don't want to show it for that, other than that even if it's true or false I want to render that component